src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -26,9 +29,23 @@ import UpdatePhone from './components/UpdatePhoneDialog';
 import styles from './PersonalInfoList.module.css';
 import AlumniUpdateForm from './components/AlumniUpdateForm';
 import CliftonStrengthsService from 'services/cliftonStrengths';
+import SLock from './Salsbury.png';
```

Comment:
Hey @ahwnsals1234, I'm guessing you created this branch from your previous work instead of starting with `develop` as your base. Because of that, you have unrelated changes in this branch. I would recommend creating a new branch from develop and using `git cherry-pick` to pull out the specific commits that you want in this pull request.

https://git-scm.com/docs/git-cherry-pick

Reply:
Hey Bennett, thank you for the feedback and sorry I'm giving you all the inconvenience. I did make a new branch from develop and made changes there. Would you prefer me to erase this PR and the branch?

Reply:
No problem! Happens to the best of us.

Yea, unless you have changes in this branch that are not anywhere else, I would recommend closing this PR and deleting the branch.